rocketc/rocketc.py

```diff
@@ -106,29 +108,39 @@ def init(self):
         else:
             return response['errorType']
 
+    def get_url_api_rocket_chat(self):
+        """
+        This method retunrs the rocketChat url service where someone can acces to API
+        """
+        settings = self.get_xblock_settings()
+        if "url_service" in settings:
+            return "{}/{}/{}".format(settings["url_service"], "api", "v1")
+
+        return "{}/{}/{}".format("http://localhost:3000", "api", "v1")
+
     def get_user_data(self):
         """
         This method initializes the user's parameters
         """
-        runtime = self.xmodule_runtime # pylint: disable=no-member
+        runtime = self.xmodule_runtime  # pylint: disable=no-member
         user = runtime.service(self, 'user').get_current_user()
         user_data = {}
         user_data["email"] = user.emails[0]
         user_data["role"] = runtime.get_user_role()
         user_data["course"] = runtime.course_id.course
         user_data["username"] = user.opt_attrs['edx-platform.username']
         user_data["anonymous_student_id"] = runtime.anonymous_student_id
-        self.user_data = user_data # pylint: disable=attribute-defined-outside-init
+        self.user_data = user_data  # pylint: disable=attribute-defined-outside-init
 
     def get_admin_data(self):
         """
         This method initializes admin's authToken and userId
         """
-        url = "{}/{}".format(URL_PREFIX, "login")
+        url = "{}/{}".format(self.url_api_rocket_chat, "login")
```

Comment:
the API Url is something like: `http://localhost:3000/api/v1`. This means, the result to url here is `http://localhost:3000/api/v1/login`. Correct?

Reply:
yes, the result is that